Routers/recipesRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const DB = require(".././model/recipesModel.js");
+
+router.get("/:id", async (req, res) => {
+  const id = req.params.id;
+  try {
+    const recipes = await DB.getChefRecipes(id);
+    res.status(200).json(recipes);
+  } catch (err) {
+    res.status(400).json(err.message);
```

Comment:
If it falls back to this error, it would actually be a 500 error.

400 is bad request used for when the client provides a lack of information or improper information.